### PR TITLE
Description of components specification in installer CR

### LINKED ISSUE #468: Provide documentation on components installation
AC:
- documentation should describe components installation mechanism

### PR DESCRIPTION
**Pull request by crabtree**
**Description**

Changes proposed in this pull request:

- Description of components specification in installer CR

**Related issue(s)**

Resolves #468

### CODE REVIEW

File: components/installer/README.md [hunk 1]
```diff
@@ -120,12 +120,20 @@ Helm installation.
...
+spec:
+    ...
+    components:
+    ...
+    - name: "remote-environments"
+      namespace: "kyma-integration"
+      release: "hmc-default"
+```
+
+In the above example the component `remote-environemnts` will be installed into the namespace `kyma-integration` using `hmc-default` as its release name. Fields `name` and `namespace` are mandatory. The name of the component is also the name of the component subdirectory in the `resources` directory. Installer assumes that component subdirectory is valid helm chart.
```
Comment (jakkab) [inline components/installer/README.md]: In the example above, the `remote-environments` component will be installed in the `kyma-integration` Namespace, using `hmc-default` as the release name. [...] Installer assumes that the component subdirectory is a valid Helm chart.

File: components/installer/README.md [hunk 2]
```diff
@@ -120,12 +120,20 @@ Helm installation.
...
-- istio
-- prometheus-operator
-- provision-bundles
-- dex
-- core
-- remote-environments
\ No newline at end of file
+By default, this tool installs predefined components. If necessary, you can specify which components to install by customizing the component list located in the [installer-cr.yaml.tpl](../../installation/resources/installer-cr.yaml.tpl) file. 
+
+To define installation of a component you need to specify component name and namespace. You can also specify helm release name for installed component (by default the name of the component will be used).
```
[the remark this answers is not in the document]
Comment (crabtree) [reply components/installer/README.md]: Done

File: components/installer/README.md (hunk 1 above)
Comment (crabtree) [reply components/installer/README.md]: Done

File: components/installer/README.md [hunk 3]
```diff
@@ -120,12 +120,20 @@ Helm installation.
...
-
-- cluster-essentials
-- istio
-- prometheus-operator
-- provision-bundles
-- dex
-- core
-- remote-environments
\ No newline at end of file
+This tool installs components specified in installation CR. Please see CR template in the [installer-cr.yaml.tpl](../../installation/resources/installer-cr.yaml.tpl) file for more details.
```
Comment (klaudiagrz) [inline components/installer/README.md]: ... ``the `Installation` CR. See the [installer-cr.yaml.tpl](../../installation/resources/installer-cr.yaml.tpl) file for more details. ``

File: components/installer/README.md [hunk 4]
```diff
@@ -120,12 +120,20 @@ Helm installation.
...
+spec:
+    ...
+    components:
+    ...
+    - name: "remote-environments"
+      namespace: "kyma-integration"
+      release: "hmc-default"
+```
+
+In the example above, the `remote-environments` component will be installed in the `kyma-integration` Namespace, using `hmc-default` as the release name.. Fields `name` and `namespace` are mandatory. The name of the component is also the name of the component subdirectory in the `resources` directory. Installer assumes that the component subdirectory is a valid Helm chart.
```
Comment (klaudiagrz) [inline components/installer/README.md]: 1. `In the example` ~above~``, the `remote-environments` component is installed``...
Also: please remove double dot at the end of this sentence.

2. ``The **name** and **namespace** fields are mandatory.``

3. `The Installer assumes`...

File: components/installer/README.md (hunk 3 above)
Comment (crabtree) [reply components/installer/README.md]: Done

File: components/installer/README.md (hunk 4 above)
Comment (crabtree) [reply components/installer/README.md]: Done

**Pull request merged**